src/common/ColourPalette.cpp

```diff
@@ -38,6 +39,10 @@ uInt32 ColourPalette::getRGB(int val) const {
     return m_palette[val];
 }
 
+void ColourPalette::getPaletteCopy(int palette_copy[256])
+{
+    memcpy(palette_copy, m_palette, 256*sizeof(int));
```

Comment:
Not really... Array decay to pointers when passed as an argument.
If you try sizeof(palette_copy) the result is 8 == the size of int*

We can take the 256 from the declaration but I though it was good as a way to tell in the header that the method expects 256.